server/routes/summary_routes.ts

```diff
@@ -61,15 +60,13 @@ export function registerSummaryAssistantRoutes(
         context.assistant_plugin.logger.error('Execute agent failed!', e);
         if (e.statusCode >= 400 && e.statusCode <= 499) {
           return res.customError({
-            body: e.body,
+            body: { message: typeof e.body === 'string' ? e.body : JSON.stringify(e.body) },
             statusCode: e.statusCode,
-            headers: e.headers,
           });
         } else {
           return res.customError({
             body: 'Execute agent failed!',
             statusCode: 500,
```

Comment:
Should we return original statusCode? As long as we don't disclose server side information, we should be fine.